app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminCheckDefaultValueDisableAutoGroupInCustomerFormActionGroup.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <!-- Check Default Value for Disable Automatic Group Changes Based on VAT ID in Customer Form -->
+    <actionGroup name="AdminCheckDefaultValueDisableAutoGroupInCustomerFormActionGroup">
+        <annotations>
+            <description>Check Default Value for Disable Automatic Group Changes Based on VAT ID in Create Customer form.</description>
+        </annotations>
+        <arguments>
+            <argument name="isChecked" type="string"/>
+        </arguments>
+
+        <amOnPage url="{{AdminNewCustomerPage.url}}" stepKey="navigateToCustomers"/>
```

Comment:
The purpose of an Action Group is to do one as less actions as possible, this way it can be reused by another tests. 
Please move the navigating to customer new page into a separate ActionGroup.

Reply:
I have just fixed it. Please check again. @eduard13

Reply:
Thank you. 
And one more small notice, please adjust the `NavigateNewCustomerActionGroup` name to follow the naming convention mentioned in the best practices guide.
`Follow naming conventions. Action Groups, Tests name should have the following format: {Assert/Storefront/Admin}{Functionality}ActionGroup.xml.`

Reply:
it is ok now @eduard13 . Thanks.